src/main/java/com/keybox/manage/action/UsersAction.java

```diff
@@ -121,17 +121,21 @@ public void validateSaveUser() {
                 || user.getFirstNm().trim().equals("")) {
             addFieldError("user.firstNm", REQUIRED);
         }
-        
-        if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
-            
-            if(!user.getPassword().equals(user.getPasswordConfirm())) {
+
+        // Password validation is only for users who are permitted to log in (ie non-passive ones)
+        if (user != null && !User.PASSIVE.equals(user.getUserType())) {
+            if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
```

Comment:
> Saving a passive user's data will always have a user entity with an empty password

No it doesn't look like that's the case in the lines I noted above. If you change a user to 'passive' it allows you to set a password, but now without the validation.